src/MEAM/pair_meam.cpp

```diff
@@ -429,13 +449,23 @@ void PairMEAM::read_global_meam_file(const std::string &globalfile)
         b1[index] = values.next_double();
         b2[index] = values.next_double();
         b3[index] = values.next_double();
+        if (msmeamflag){
```

Comment:
In the LAMMPS programming style conventions there is always a blank before opening braces.